dts/bindings/gpio/altr,pio.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Altera PIO node
+
+compatible: "altr,pio"
```

Comment:
It would be nice if this matched the upstream binding for the same IP: https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/gpio/gpio-altera.txt